open_event/api/sponsors.py

```diff
@@ -39,6 +40,13 @@ def get(self, event_id, sponsor_id):
         """Fetch a sponsor given its id"""
         return DAO.get(event_id, sponsor_id)
 
+    @requires_auth
+    @api.doc('delete_sponsor')
+    @api.marshal_with(SPONSOR)
+    def delete(self, event_id, sponsor_id):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/triplez/open-event-orga-server/file/2716743260/issues/source?bid=3233864&fileBranchId=3345807#l46)